### DISCUSSION TITLE
Error Engine node

### DISCUSSION THREAD

**Discussion by killoldesai** [tezjs/tezjs#132] [General]
Hi

getting following error 

PS E:\work\no> npm run develop

> no@0.0.0 develop
> cross-env FORCE_COLOR=1 npm-run-all -l -p develop:*

[develop:backend ]
[develop:backend ] > no@0.0.0 develop:backend
[develop:backend ] > yarn --cwd backend develop
[develop:backend ]
[develop:frontend]
[develop:frontend] > no@0.0.0 develop:frontend
[develop:frontend] > wait-on http://localhost:1337/admin && yarn --cwd frontend dev
[develop:frontend]
yarn run v1.22.19
error blog-tezjs@0.1.0: The engine "node" is incompatible with this module. Expected version ">=12.x.x <=16.x.x". Got "18.12.1"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[develop:backend ] error Commands cannot run with an incompatible environment.
ERROR: "develop:backend" exited with 1.
PS E:\work\no>

please help

**Comment by ajayojha** [comment]
Node version is incompatible, please update with the latest version of nodejs

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: killoldesai ***@***.***>
Sent: Monday, November 28, 2022 9:45:37 PM
To: tezjs/tezjs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [tezjs/tezjs] Error Engine node (Discussion #132)


Hi

getting following error

PS E:\work\no> npm run develop

***@***.*** develop
cross-env FORCE_COLOR=1 npm-run-all -l -p develop:*

[develop:backend ]
[develop:backend ] > ***@***.*** develop:backend
[develop:backend ] > yarn --cwd backend develop
[develop:backend ]
[develop:frontend]
[develop:frontend] > ***@***.*** develop:frontend
[develop:frontend] > wait-on http://localhost:1337/admin && yarn --cwd frontend dev
[develop:frontend]
yarn run v1.22.19
error ***@***.***: The engine "node" is incompatible with this module. Expected version ">=12.x.x <=16.x.x". Got "18.12.1"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[develop:backend ] error Commands cannot run with an incompatible environment.
ERROR: "develop:backend" exited with 1.
PS E:\work\no>

please help

**Reply by killoldesai** [reply]
its already latest please again read

error blog-tezjs@0.1.0: The engine "node" is incompatible with this module. Expected version ">=12.x.x <=16.x.x". **Got "18.12.1"**

**Comment by ajayojha** [comment]
Install between this - Expected version ">=12.x.x <=16.x.x"

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
